### ISSUE TITLE
Add metric scrape_target_response_size ，to record the size of the return body of the scrape target

### ISSUE THREAD

**Issue by BegoniaGit** [prometheus/prometheus#9520]
## Proposal
**Use case. Why is this important?**
In some of our large-scale deployment scenarios, we need to evaluate the network bandwidth by querying the scrape target return body size.
 I wish I could submit the Merge Request to improve this functionality。
*“Nice to have” is not a good use case. :)*

**Comment by roidelapluie** [comment]
Yes, you could start with #7722 and add it behind the extra scrape metrics feature flag.
